downloadDirectory/src/main/java/org/reactome/release/downloadDirectory/GenerateGOAnnotationFile/GOAGeneratorUtilities.java

```diff
@@ -131,31 +129,31 @@ private static String getSecondaryIdentifier(GKInstance referenceEntityInst) thr
 
     /**
      * Checks that the protein is not from an excluded microbial taxon.
-     * @param taxonIdentifier -- Protein's Species' CrossReference identifier.
-     * @return
+     * @param taxonIdentifier -- String, Protein's Species' CrossReference identifier.
+     * @return -- true if the taxonIdentifier is found in the microbialSpeciesToExclude array, false if not.
      */
     public static boolean excludedMicrobialSpecies(String taxonIdentifier) {
         return microbialSpeciesToExclude.contains(taxonIdentifier);
     }
 
     /**
      * Checks that the GO accession is not for Protein Binding. These don't receive a GO annotation since they require an "IPI" evidence code.
-     * @param goAccession -- GO accession string.
-     * @return
+     * @param goAccession -- Object, GO accession string.
```

Comment:
Just wondering - why is this parameter's type Object instead of String?

Reply:
I think that originally I did this to reduce clutter from the method call. After the refactors though it isn't so bad so maybe I'll just cast or use `toString()` for the method call